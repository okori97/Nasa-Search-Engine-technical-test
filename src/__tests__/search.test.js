import React from "react";
import Search from "../components/search";
import { render } from "@testing-library/react";

describe("Search", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Search />);

    expect(asFragment()).toMatchSnapshot();
  });

  it("has placeholder text", () => {
    const { getByPlaceholderText } = render(<Search />);

    expect(getByPlaceholderText("Search image..")).toHaveClass(
      "form__searchField"
    );
  });
});
