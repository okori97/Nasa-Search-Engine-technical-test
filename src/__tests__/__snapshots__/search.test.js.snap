// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<DocumentFragment>
  <form
    class="form"
  >
    <input
      class="form__searchField"
      placeholder="Search image.."
      type="text"
    />
    <button
      class="form__button"
      type="submit"
    >
      <span
        class="material-symbols-outlined"
      >
        search
      </span>
    </button>
  </form>
</DocumentFragment>
`;
